pylxd/models/operation.py

```diff
@@ -40,8 +40,28 @@ def get(cls, client, operation_id):
         response = client.api.operations[operation_id].get()
         return cls(_client=client, **response.json()['metadata'])
 
-    def __init__(self, **kwargs):
+    @classmethod
+    def all(cls, client):
+        """List of operations"""
+        response = client.api.operations.get()
+
+        operations = []
+        for op_id in response.json()['metadata']['running']:
+            operations.append(cls.get(client, op_id))
+        return operations
+
+    @property
+    def api(self):
+        return self._client.api.operations[self.id]
+
+    @property
+    def class_(self):
+        """ Avoid class as reserved keyworkds """
+        return getattr(self, "class")
+
+    def __init__(self, client=None, **kwargs):
```

Comment:
You'll need to re-base your PR, please, as there was a bug fixed in the `operation.py` file; please see the master branch.